plugins/network-elements/nuage-vsp/src/net/nuage/vsp/acs/client/NuageVspSyncClient.java

```diff
@@ -21,7 +21,7 @@
 
 public interface NuageVspSyncClient {
 
-    public void syncWithNuageVsp(String nuageVspEntity) throws Exception;
+    void syncWithNuageVsp(String nuageVspEntity) throws Exception;
```

Comment:
This ``throws`` is too broad.  Please enumerate the expected checked exceptions to be thrown.